src/pixelator/plot/__init__.py

```diff
@@ -392,8 +406,10 @@ def _get_color_values(mark, filtered_coordinates, log_scale, vmax):
 def plot_2d_graph(
     pxl_data: PixelDataset,
     component: Union["str", list],
```

Comment:
This type hint does not seem to match with the code

```suggestion
    component: Iterable[str]
```

Reply:
Should it be Union[str, Iterable[str]]? Because for the case where the user provides only one component we have:
if isinstance(component, str):
        component = [component]